redis.conf

```diff
@@ -1479,23 +1479,25 @@ aof-timestamp-enabled no
 #
 # shutdown-timeout 10
 
-################################ LUA SCRIPTING  ###############################
+########################## LONG BLOCKING COMMANDS #############################
 
-# Max execution time of a Lua script in milliseconds.
+# Maximum time in milliseconds for EVAL scripts, Functions and in some cases
+# Modules before redis can start processing or rejecting other clients.
 #
-# If the maximum execution time is reached Redis will log that a script is
-# still in execution after the maximum allowed time and will start to
-# reply to queries with an error.
+# If the maximum execution time is reached Redis will start to reply to most
+# commands with a BUSY error.
 #
-# When a long running script exceeds the maximum execution time only the
-# SCRIPT KILL and SHUTDOWN NOSAVE commands are available. The first can be
-# used to stop a script that did not yet call any write commands. The second
-# is the only way to shut down the server in the case a write command was
-# already issued by the script but the user doesn't want to wait for the natural
-# termination of the script.
+# In this state Redis will only allow a handful of commands to be executed.
+# For instance, SCRIPT KILL, FUNCTION KILL, SHUTDOWN NOSAVE and possibly some
+# module specific commands.
+#
+# SCRIPT KILL and FUNCTION KILL will only be able to stop a script that did not
+# yet call any write commands, so SHUTDOWN NOSAVE may be the only way to stop
+# the server in the case a write command was already issued by the script but
+# the user doesn't want to wait for the natural termination of the script.
 #
 # Set it to 0 or a negative value for unlimited execution without warnings.
-lua-time-limit 5000
+busy-reply-threshold 5000
```

Comment:
Maybe adding new special configs is better, avoid breaking change since most users are using the old config `lua-time-limit`.

Reply:
it's not a breaking change, since there's an alias.
but maybe in this case we want to still mention the old config in the documentation (so whoever sees it in some code can find it's docs).
i'll do that.